pages/_app.js

```diff
@@ -1,12 +1,87 @@
 import GlobalStyle from "../styles";
-import { bags } from "../components/List/data";
-import List from "./List";
+import { airlines } from "../lib/data";
+import { useState } from "react";
+import { calculateVolume } from "@/utils";
 
 export default function App({ Component, pageProps }) {
+  const [unitSystem, setUnitSystem] = useState("metric");
+  const [sortedAirlines, setSortedAirlines] = useState(airlines);
+
+  function handleUnitSystemChange(option) {
+    if (option === "metric") {
+      setUnitSystem("metric");
+    } else if (option === "imperial") {
+      setUnitSystem("imperial");
+    }
+  }
```

Comment:
since option is either "metric" or "imperial" you can pass it to the setUnitSystem direcly
```suggestion
  function handleUnitSystemChange(option) {
    setUnitSystem(option);
  }
```

Reply:
Makes totally sense, applied.